arrow-site/docs/docs/arrow/core/continuations/README.md

```diff
@@ -0,0 +1,202 @@
+# Semantics of Structured Concurrency + Effect<R, A>
```

Comment:
This is not the documentation we want, this was part of the design process/documentation.
What we want instead is the [README.MD](https://github.com/nomisRev/Continuation/blob/main/README.md)

I'm thinking we should make it part of the KDoc on top of `Effect`, and link the page to `Effect` as well.

Currently, a lot of users have a hard time understanding the difference between `Effect` and `EagerEffect` and when or why they should be using `either { }` vs `either.eager { }`. This is currently completely undocumented inside `arrow-continuations`/`arrow.core.computations`.

I think we should add proper documentation in this PR before we merge, otherwise, it's going to be dropped to the bottom of the backlog again.

Reply:
Updated most places I was aware of, can I get another review